src/manage_pg.c

```diff
@@ -1114,24 +1114,32 @@ manage_create_sql_functions ()
              "         END;"
              "$$ LANGUAGE SQL;");
 
-      sql ("CREATE OR REPLACE FUNCTION task_last_report (integer)"
-           " RETURNS integer AS $$"
-           /* Get the report from the most recently completed invocation of task. */
-           "  SELECT id FROM reports WHERE task = $1 AND scan_run_status = %u"
-           "  ORDER BY date DESC LIMIT 1;"
-           "$$ LANGUAGE SQL;",
-           TASK_STATUS_DONE);
-
-      sql ("CREATE OR REPLACE FUNCTION task_second_last_report (integer)"
-           " RETURNS integer AS $$"
-           /* Get report from second most recently completed invocation of task. */
-           "  SELECT id FROM reports WHERE task = $1 AND scan_run_status = %u"
-           "  ORDER BY date DESC LIMIT 1 OFFSET 1;"
-           "$$ LANGUAGE SQL;",
-           TASK_STATUS_DONE);
+      /* column date in table reports was renamed to creation_time in version 245 */
+      if (current_db_version >= 245) {
```

Comment:
I'd prefer it if you either removed the braces or used the format we use elsewhere:
```c
if (current_db_version >= 245)
  {
    sql ("...");
  }
```